AUTHORS

```diff
@@ -0,0 +1,7 @@
+         Petr Svarny   <petr.s@praguecollege.cz>
```

Comment:
Our lector is a great person, but I doubt we will be able to get evaluated by him if we list him as an author :)